TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/CommonJourneyPage.cs

```diff
@@ -0,0 +1,43 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeachingRecordSystem.SupportUi;
+using TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.EditInduction;
+
+public abstract class CommonJourneyPage : PageModel
+{
+    protected TrsLinkGenerator _linkGenerator;
+    public JourneyInstance<EditInductionState>? JourneyInstance { get; set; }
+    public virtual InductionStatus InductionStatus
+    {
+        get { return JourneyInstance!.State.InductionStatus; }
+        set { }
```

Comment:
Does this need a setter?

Reply:
I've moved it back out of here and into each page that needs it